config/locales/js-en.yml

```diff
@@ -1287,6 +1287,7 @@ en:
         include_all_selected: 'Include all sub-projects is currently selected.'
         current_project: 'This is the current project you are in.'
         does_not_match_search: 'Project does not match the search criteria.'
+      no_results: 'No project matching your search criteria'
```

Comment:
I'd rather upgrade this group of words to a sentence: `No project matches your search criteria.`

IMHO it looks weird when there is no full stop in this place.

Reply:
@marcalcobe Any thoughts on that?

Reply:
@HDinger and @Kharonus looks good to me